docs/plugins.rst

```diff
@@ -70,6 +86,33 @@ which you map the signals to your plugin logic. Let's take a simple example::
     your ``register`` callable or they will be garbage-collected before the
     signal is emitted.
 
+Namespace Plugin structure
```

Comment:
`Namespace Plugin structure` → `Namespace plugin structure` (to maintain consistent capitalization)